webapp/app/[locale]/tunnel/_components/form.tsx

```diff
@@ -45,10 +51,18 @@ export const FormContent = function ({
     toToken,
   } = tunnelState
 
+  const [customTokenAddress] = useCustomTokenAddress()
+  const hemi = useHemi()
   const t = useTranslations('tunnel-page')
 
   const showToast = useToastIfNotConnectedTo(fromNetworkId)
 
+  const evmTunneling =
+    typeof fromNetworkId === 'number' && typeof toNetworkId === 'number'
+
+  const l1ChainId = fromNetworkId === hemi.id ? toNetworkId : fromNetworkId
+  const l2ChainId = fromNetworkId === l1ChainId ? toNetworkId : fromNetworkId
```

Comment:
Current logic is kind of difficult to read. Consider these options instead:

```suggestion
  const l1ChainId = fromNetworkId === hemi.id ? toNetworkId : fromNetworkId
  const l2ChainId = fromNetworkId !== hemi.id ? toNetworkId : fromNetworkId
```

or

```suggestion
  const [l1ChainId, l2ChainId] = fromNetworkId === hemi.id
    ? [toNetworkId, fromNetworkId] 
    : [fromNetworkId, toNetworkId]
```

Moreover, in which case `l2ChainId` is different from `hemi.id`?